packages/web-service/src/pages/returns/why-nil.njk

```diff
@@ -33,13 +33,19 @@
             errorMessage: { text: 'Select an option' } if error['why-nil'],
             items: [
               {
+                value: data.THE_DEVELOPMENT_WORK_DID_NOT_HAPPEN,
                 text: "The development work did not happen",
+                checked: data.whyNil === data.THE_DEVELOPMENT_WORK_DID_NOT_HAPPEN
               },
               {
+                value: data.THE_SETT_WAS_NOT_IN_ACTIVE_USE_BY_BADGERS,
                 text: "The sett was not in active use by badgers",
+                checked: data.whyNil === data.THE_SETT_WAS_NOT_IN_ACTIVE_USE_BY_BADGERS
               },
               {
+                value: data.OTHER,
                 text: "Other",
+                checked: payload['why-nil'] === '452120002' or data.whyNil === data.OTHER,
```

Comment:
Maybe you should pass the number from the getData, rather than hardcoding it here?

Reply:
will sort it in the coming PR thanks @kealanparr